### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [AlmaLinux Foundation/almalinux.org](https://hosted.weblate.org/projects/almalinux/website-backend/).

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by bennyvasquez** [comment]
@codyro I think I fixed that string in a different PR. Can we compare this to a future version? or should we just close this one and then do a fresh request from Weblate?

**Comment by codyro** [comment]
> @codyro I think I fixed that string in a different PR. Can we compare this to a future version? or should we just close this one and then do a fresh request from Weblate?

Which string are you referring to? I think it'd be easiest if we updated the incorrect string in Weblate.

File: i18n/es.json [hunk 1]
```diff
@@ -566,5 +566,57 @@
...
+    "Can't make it?": "¿No puede asistir?",
+    "Meet the Experts": "Conozca a los expertos",
+    "December 9th, 2023": "9 de diciembre de 2023",
+    "\"AlmaLinux Day Tokyo 2023\" is the world's first official AlmaLinux OS Foundation event. This event will be a gathering of some of the most engaged people in the AlmaLinux community, as well as technology partners from all over the world. This is the best place to engage and meet all of the people": "\"Día de AlmaLinux en Tokio 2023\" es el primer evento oficial de la Fundación AlmaLinux OS en el mundo. Este evento será una reunión de algunas de las personas más comprometidas en la comunidad AlmaLinux, así como socios tecnológicos de todo el mundo. Este es el mejor lugar para participar y conocer a todas las personas",
```
Comment (bennyvasquez) [inline i18n/es.json:593]:
```suggestion
    "AlmaLinux Day Tokyo 2023 is the world's first official AlmaLinux OS Foundation event. This event will be a gathering of some of the most engaged people in the AlmaLinux community, as well as technology partners from all over the world. This is the best place to engage and meet all of the people": "Día de AlmaLinux en Tokio 2023 es el primer evento oficial de la Fundación AlmaLinux OS en el mundo. Este evento será una reunión de algunas de las personas más comprometidas en la comunidad AlmaLinux, así como socios tecnológicos de todo el mundo. Este es el mejor lugar para participar y conocer a todas las personas",
```

File: i18n/sv.json [hunk 2]
```diff
@@ -558,5 +558,57 @@
...
+    "Can't make it?": "Kan du inte komma?",
+    "Meet the Experts": "Möt experterna",
+    "December 9th, 2023": "9:e December 2023",
+    "\"AlmaLinux Day Tokyo 2023\" is the world's first official AlmaLinux OS Foundation event. This event will be a gathering of some of the most engaged people in the AlmaLinux community, as well as technology partners from all over the world. This is the best place to engage and meet all of the people": "\"AlmaLinux Day Tokyo 2023\" är världens första officiella AlmaLinux OS stiftelse evenemang. Detta evenemang kommer att vara en sammankomst av några av de mest engagerade människorna i AlmaLinux gemenskapen, såväl som teknikpartners från hela världen. Detta är det bästa stället att engagera sig och träffa alla människor",
```
Comment (bennyvasquez) [inline i18n/sv.json:585]:
```suggestion
    "AlmaLinux Day Tokyo 2023 is the world's first official AlmaLinux OS Foundation event. This event will be a gathering of some of the most engaged people in the AlmaLinux community, as well as technology partners from all over the world. This is the best place to engage and meet all of the people": "AlmaLinux Day Tokyo 2023 är världens första officiella AlmaLinux OS stiftelse evenemang. Detta evenemang kommer att vara en sammankomst av några av de mest engagerade människorna i AlmaLinux gemenskapen, såväl som teknikpartners från hela världen. Detta är det bästa stället att engagera sig och träffa alla människor",
```

**Review by bennyvasquez — APPROVED** [review]

**Pull request merged**